Sources/PostgresNIO/Connection/PostgresConnection.swift

```diff
@@ -530,6 +541,12 @@ extension PostgresConnection {
             throw error // rethrow with more metadata
         }
     }
+
+    private func write<T, Value>(_ any: T, cascadingFailureTo promise: EventLoopPromise<Value>) {
```

Comment:
we don't need T here. Just make it `HandlerTask`

Reply:
done